app/models/SubmissionProcessingResult.scala

```diff
@@ -20,6 +20,17 @@ sealed trait SubmissionProcessingResult
 
 object SubmissionProcessingResult {
 
+  def format(result: SubmissionProcessingResult) = result match {
+    case SubmissionSuccess =>
+      "success"
+    case SubmissionFailureInternal =>
+      "failure-internal"
+    case SubmissionFailureExternal =>
+      "failure-external"
+    case SubmissionFailureRejected(responseBody) =>
+      "rejected"
```

Comment:
I think previously the entire responseBody was `toString`ed into the metric name so I added this